### ISSUE TITLE
No training loss in tensorboard in tpu_lm_finetuning.py

### ISSUE THREAD

**Issue by vbogach** [mgrankin/ru_transformers#29]
https://github.com/mgrankin/ru_transformers/blob/766343f3ed121ca7c4583c1d0cfaa4a390e88db8/tpu_lm_finetuning.py#L433

summary_write(...) is called for the learning rate, but not for training loss.
